### PR TITLE
Search bar/merge from main

### PR DESCRIPTION
**Pull request by Koyamot**
• Fixes background image search bar
• Styled search bar

### CODE REVIEW

**Review by stevenwaldron — APPROVED** [review]
looks good!

File: src/components/common/SearchForm.js [hunk 1]
```diff
@@ -62,41 +62,23 @@ const SearchForm = ({ fetchCityData }) => {
...
-            style={SearchStyle}
-            value={searchValue.city}
-            onChange={handleChange}
-          /> */}
           <Dropdown overlay={menu} trigger={['click']}>
             <a
               className="ant-dropdown-link"
               onClick={
                 e => e.preventDefault()
                 // onSubmit()
```
Comment (cdlafortune) [inline src/components/common/SearchForm.js:70]: please remove the commented-out code in this file

**Review by cdlafortune — CHANGES_REQUESTED** [review]
the PR description could be more detailed - there are a LOT of changed lines on this PR, and I'm not 100% sure why the package.json and package-lock were removed.
this could cause major problems for teammates if this was merged into main as it is since those files were not replaced. 
seeing some linting errors for unused variables and such -- review and remove unneeded code

**Review by cdlafortune — APPROVED** [review]
per our conversation during stand up, the deleted files were to resolve merge conflicts and the unused variables will be used now that we have access to the DS API

**Pull request merged**